### DISCUSSION TITLE
Best way to ignore hidden points from gltf

### DISCUSSION THREAD

**Discussion by bscott711** [pyvista/pyvista#1709] [Q&A]
I am having an issue using gltf files that have hidden points. The output from pl1 is as expected (a single surface), but I am trying to obtain the mesh vertices and faces for additional analysis, and the all_points_0 contains all of the hidden points. Is there an easy way to ignore these points?

```py
pl1 = pv.Plotter()
pl2 = pv.Plotter()
mesh_0 = pv.MultiBlock('example.glb')
all_points_0 = mesh_0.combine().points
pl1.add_mesh(mesh_0)
pl2.add_mesh(all_points_0)
pl1.show()
pl2.show()
```
![image](https://user-images.githubusercontent.com/40175801/136241851-a10b935b-ca24-4d96-bb30-43a9f4320277.png)

![image](https://user-images.githubusercontent.com/40175801/136241906-dbe5b89c-b701-48a1-9850-025dced1a412.png)

**Comment by banesullivan** [comment]
Can you share the data?
